src/game/server/gamecontroller.cpp

```diff
@@ -70,12 +70,13 @@ void IGameController::DoActivityCheck()
 				switch(g_Config.m_SvInactiveKick)
 				{
 				case 0:
```

Comment:
Bug? case 0 does nothing? If this is intended, update the description of the command instead.

Reply:
case 0 does the same as case 1 for players

Reply:
0=move player to spectator
1=move player to spectator and kick spectator
2=move player to free spectator slot/kick and kick spectator
3=kick either player or spectator

Reply:
Oh, nevermind, read too fast. My bad.